### PR TITLE
KRACOEUS-8845: Adding ScaleTwo and ScaleThree decimal support.

### PR DESCRIPTION
**Pull request by gathreya**

### CODE REVIEW

**Comment by gathreya** [comment]
PLEASE do not merge before https://github.com/KualiCo/rice-kc/pull/2 is merged!! Code will not compile.

File: coeus-impl/src/main/java/org/kuali/coeus/sys/impl/controller/TypeUtilsAdditionServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.kuali.coeus.sys.api.model.ScaleThreeDecimal;
+import org.kuali.coeus.sys.api.model.ScaleTwoDecimal;
+import org.kuali.coeus.sys.framework.controller.TypeUtilsAdditionService;
+import org.kuali.rice.core.api.util.type.TypeUtils;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component("typeUtilsAdditionService")
+public class TypeUtilsAdditionServiceImpl implements TypeUtilsAdditionService {
```
Comment (leoherbie) [inline coeus-impl/src/main/java/org/kuali/coeus/sys/impl/controller/TypeUtilsAdditionServiceImpl.java]: I wouldn't call this a "service".  Maybe an "Initializer" or something.

**Comment by gathreya** [comment]
Made changes requested but please merge rice changes first.

**Comment by blackcathacker** [comment]
Looks like the rice change was merged and built a few hours ago. Are we ready to do this now?

**Comment by gathreya** [comment]
please merge @blackcathacker

**Pull request merged**